standard/types.md

```diff
@@ -836,28 +831,27 @@ The ***default null state*** of an expression is determined by its type:
 
 - The default null state of a nullable reference type is null.
 - The default null state of a non-nullable reference type is not null.
-- The default null state of a nullable oblivious reference type is not null.
 
-A warning might result if a variable of a non-nullable type has a potential `null` value assigned to it. However, oblivious and nullable variables can have `null` values freely assigned to them.
+A warning might result if a variable of a non-nullable type has a potential `null` value assigned to it. Nullable variables can have `null` values freely assigned to them.
 
 > *Note*: Assigning a maybe null expression to a variable `v` of a non-nullable reference type sets the null state of `v` to maybe null. *end note*
 
 Nullability is determined, as follows:
 
 - A non-nullable value type `S` is always non-nullable
 - A nullable value type `S?` is always nullable
-- A reference type `C` in a disabled nullable annotation context is oblivious
+- A reference type `C` in a disabled nullable annotation context is nullable, however its default null state is *not-null*.
 - A reference type `C` in an enabled nullable annotation context is non-nullable
 - A nullable reference type `C?` is nullable; however, a warning shall be generated in a disabled nullable annotation context
 
 With a type parameter, any constraints are taken into account:
 
 - A type parameter `T` where all constraints (if any) are either nullable types or the `class?` constraint, is `nullable`
-- A type parameter `T` where at least one constraint is either oblivious or non-nullable, or one of the constraints `struct`,  `class`, or `notnull`, is
+- A type parameter `T` where at least one constraint is non-nullable, or one of the constraints `struct`,  `class`, or `notnull`, is
```

Comment:
Here's another example of where we refer to "oblivious" without really defining it.

Reply:
Should be fixed.

Reply:
Still says oblivious in the line below...